### PR TITLE
Prototype `PROTO.browsingContext.findElement`.

### PR DESCRIPTION
**Pull request by sadym-chromium**
+ implement (with some suggestions) NodeRemoteValue serialization.

### CODE REVIEW

File: src/bidiMapper/scripts/eval.es [hunk 1]
```diff
@@ -70,6 +70,44 @@
...
+            nodeType: value.nodeType,
+            childNodeCount: value.childNodes.length
+          }
+          if(value.nodeValue!==null)
+            result.nodeValue = value.nodeValue;
+
+          // TODO: current implementation serializes `childNodes` instead of `children`. Decide
+          // which way is better and fix specification or implementation.
+          result.children = [];
+          for(var i=0; i<value.childNodes.length; i++){
```
Comment (bwalderman) [inline src/bidiMapper/scripts/eval.es:90]:
```suggestion
          for (let childNode of value.childNodes.values()) {
```

File: src/bidiMapper/scripts/eval.es [hunk 2]
```diff
@@ -70,6 +70,44 @@
...
+            const c = serialize(value.childNodes.item(i), maxDepth-1);
+            result.children.push(c);
+          }
+
+          if(value instanceof Element){
+            result.attributes = {};
+            for(var i = 0; i< value.attributes.length; i++){
```
Comment (bwalderman) [inline src/bidiMapper/scripts/eval.es:97]: Nit: Prefer `let`.

File: src/bidiMapper/scripts/eval.es [hunk 3]
```diff
@@ -70,6 +70,44 @@
...
+            childNodeCount: value.childNodes.length
+          }
+          if(value.nodeValue!==null)
+            result.nodeValue = value.nodeValue;
+
+          // TODO: current implementation serializes `childNodes` instead of `children`. Decide
+          // which way is better and fix specification or implementation.
+          result.children = [];
+          for(var i=0; i<value.childNodes.length; i++){
+            const c = serialize(value.childNodes.item(i), maxDepth-1);
```
Comment (bwalderman) [inline src/bidiMapper/scripts/eval.es:91]:
```suggestion
            const c = serialize(childNode, maxDepth-1);
```

**Review by bwalderman — APPROVED** [review]
LGTM with suggestions. Thanks!

File: src/bidiMapper/scripts/eval.es (hunk 3 above)
Comment (sadym-chromium) [reply src/bidiMapper/scripts/eval.es]: done

File: src/bidiMapper/scripts/eval.es (hunk 2 above)
Comment (sadym-chromium) [reply src/bidiMapper/scripts/eval.es]: done

File: src/bidiMapper/scripts/eval.es (hunk 1 above)
Comment (sadym-chromium) [reply src/bidiMapper/scripts/eval.es]: done

**Pull request merged**